inst/qml/RobustBayesianMetaAnalysis.qml

```diff
@@ -0,0 +1,2001 @@
+						Layout.preferredWidth: 140 * preferencesModel.uiScale
+						DropDown
+						{
+							id: typeMuItem
+							name: "type"
+							useExternalBorder: true
+							values:
+							[
+								{ label: qsTr("Normal(μ,σ)"),			value: "normal"},
+								{ label: qsTr("Student-t(μ,σ,v)"),		value: "t"},
+								{ label: qsTr("Cauchy(x₀,θ)"),			value: "cauchy"},
+								{ label: qsTr("Gamma(α,β)"),			value: "gamma_ab"},
+								{ label: qsTr("Gamma(k,θ)"),			value: "gamma_k0"},
+								{ label: qsTr("Inverse-Gamma(α,β)"),	value: "invgamma"},
+								{ label: qsTr("Spike(x₀)"),				value: "spike"},
+								{ label: qsTr("Uniform(a,b)"),			value: "uniform"}
+							]
+						}
+					}
+
+					Row
+					{
+						spacing: 4 * preferencesModel.uiScale
+						Layout.preferredWidth: 160 * preferencesModel.uiScale
+						FormulaField
+						{
+							label:				"μ "
+							name:				"parMean"
+							visible:			typeMuItem.currentValue === "normal"		||
+												typeMuItem.currentValue === "t"
+							value:				"0"
+							inclusive:			JASP.None
+							fieldWidth: 		40 * preferencesModel.uiScale
+							useExternalBorder:	false
+							showBorder: 		true
+						}
+						FormulaField
+						{
+							label:				"x₀"
+							name:				"parLocation"
+							visible:			typeMuItem.currentValue === "cauchy"	||
+												typeMuItem.currentValue === "spike"
+							value:				"0"
+							inclusive:			JASP.None
+							fieldWidth: 		40 * preferencesModel.uiScale
+							useExternalBorder:	false
+							showBorder: 		true
+						}
+						FormulaField
+						{
+							label:				"σ"
+							name:				"parScale"
+							visible:			typeMuItem.currentValue === "normal"		||
+												typeMuItem.currentValue === "t"
+							value:				"1"
+							min:				0
+							inclusive:			JASP.None
+							fieldWidth: 		40 * preferencesModel.uiScale
+							useExternalBorder:	false
+							showBorder: 		true
+						}
+						FormulaField
+						{
+							label:				"k "
+							name:				"parShape"
+							visible:			typeMuItem.currentValue === "gamma_k0"
+							value:				"1"
+							min:				0
+							inclusive:			JASP.None
+							fieldWidth: 		40 * preferencesModel.uiScale
+							useExternalBorder:	false
+						}
+						FormulaField
+						{
+							label:				"θ"
+							name:				"parScale2"
+							visible:			typeMuItem.currentValue === "cauchy"	||
+												typeMuItem.currentValue === "gamma_k0"
+							value:				"1"
+							min:				0
+							inclusive:			JASP.None
+							fieldWidth: 		40 * preferencesModel.uiScale
+							useExternalBorder:	false
+							showBorder: 		true
+						}
+						FormulaField
+						{
+							label:				"ν"
+							name:				"parDF"
+							visible:			typeMuItem.currentValue === "t"
+							value:				"2"
+							min:				1
+							inclusive:			JASP.MinOnly
+							fieldWidth: 		40 * preferencesModel.uiScale
+							useExternalBorder:	false
+							showBorder: 		true
+						}
+						FormulaField
+						{
+							label:				"α "
```

Comment:
because the point and Cauchy prior distribution have x[0] as a first parameter, this makes the labels and boxes seem more aligned if two different distributions are selected